docs/GetStarted/setup.md

```diff
@@ -123,25 +123,23 @@ Set the shared secret used in Kong for JWT authentication.
     !!! success
         ``` bash
         IMAGE                               STATUS                        NAMES
-        intel/oact-rps:v2.2.0               Up 2 minutes (healthy)        open-amt-cloud-toolkit_rps_1      
+        intel/oact-rps:latest               Up 2 minutes (healthy)        open-amt-cloud-toolkit_rps_1      
```

Comment:
are you able to use the mkdocs versions here to plug it in?

Reply:
closing since irrelevant as latest is correct